sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -1217,7 +1230,7 @@ class AdaptiveQueryExecSuite
                 assert(planInfo.nodeName == "AdaptiveSparkPlan")
                 assert(planInfo.children.size == 1)
                 assert(planInfo.children.head.nodeName ==
-                  "Execute CreateDataSourceTableAsSelectCommand")
+                  "Execute InsertIntoHadoopFsRelationCommand")
```

Comment:
shall we check 2 items? One is `CreateDataSourceTableAsSelectCommand` and the other is `InsertIntoHadoopFsRelationCommand`